web/src/pages/starter-kits/general.tsx

```diff
@@ -0,0 +1,158 @@
+import { NavBar } from "@/components/navbar/NavBar";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
+import { Card } from "@/components/starter-kits/Card";
+import { StepInfo } from "@/components/starter-kits/StepInfo";
+import { getMergedConfig } from "@/contexts/configContext";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { buildUserRoadmap } from "@/lib/roadmap/buildUserRoadmap";
+import { Roadmap, Task } from "@/lib/types/types";
+import analytics from "@/lib/utils/analytics";
+import { ProfileData, createEmptyProfileData } from "@businessnjgovnavigator/shared/index";
+import type { InferGetStaticPropsType } from "next";
+import { useRouter } from "next/router";
+import { ReactElement } from "react";
+
+const General = (props: InferGetStaticPropsType<typeof getStaticProps>): ReactElement => {
+  const router = useRouter();
```

Comment:
[dust] I wonder if we can drop the router and route manually, either by putting the buttons inside links or by setting the window location. It's clearly not affecting the starter kit, as the page's HTML is being rendered as expected, but it's one fewer frontend JS-based component here. Do you see a benefit, or do the benefits of using the router outweigh the theoretical benefits of dropping it?

Reply:
Looks like was dropped during the industry specific starter kit feature implementation 🚀